server/migrator/Importer.java

```diff
@@ -139,6 +141,7 @@ public void run() {
         insertMissingOwnerships();
         insertMissingRolePlayers();
         commit();
+        tx.close();
```

Comment:
The recommended way of closing Grakn Transactions in Java is using try-with-resources:
```java
try (tx = session.transaction(Arguments.Transaction.Type.WRITE)) {
   ...
}
```

Reply:
Actually `commit` will also `close` it, I will just delete that line.

Reply:
Ah I see that now. Would be nice if we could refactor it to be a local variable but... if it's a lot of work then don't worry about it.